webapp/src/webapp/connections/views/connection_connect.cljs

```diff
@@ -149,26 +183,29 @@
     [:> Skeleton {:height "60px"}]]])
 
 (defn- failure [connection]
-  [:main
-   [:header {:class "mb-2"}
-    [:> Heading {:size "6" :as "h2"}
-     "Hoop Access"]]
+  (let [connection-data (-> connection :data)]
+    [:main
+     [:header {:class "mb-2"}
+      [:> Heading {:size "6" :as "h2"}
+       "Hoop Access"]]
 
-   [:section {:class "my-regular"}
-    [:span {:class "text-sm text-gray-800"}
-     "Download and open Hoop Desktop App to have direct access to this connection."]]
+     [:section {:class "my-regular"}
+      [:span {:class "text-sm text-gray-800"}
+       "Download and open Hoop Desktop App to have direct access to this connection."]]
 
-   [:section {:class "flex justify-end gap-3"}
-    [:> Button {:size "2"
-                :variant "outline"
-                :color "gray"
-                :on-click #(js/window.open "https://install.hoop.dev")}
-     [:> Download {:size 16}]
-     " Download Desktop App"]
-    [:> Button {:size "2"
-                :on-click (fn []
-                            (rf/dispatch [:connections->start-connect (-> connection :data :connection_name)]))}
-     "Try again"]]])
+     [:section {:class "flex justify-end gap-3"}
+      [:> Button {:size "2"
+                  :variant "outline"
+                  :color "gray"
+                  :on-click #(js/window.open "https://install.hoop.dev")}
+       [:> Download {:size 16}]
+       " Download Desktop App"]
+      [:> Button {:size "2"
+                  :on-click (fn []
+                              (rf/dispatch [:connections->start-connect {:type (:connection_type connection-data)
+                                                                         :subtype (:connection_subtype connection-data)
```

Comment:
These are only read only fields, you don't need to send them.

Reply:
done